composer.json

```
@@ -140,8 +140,8 @@
     "codeception/module-symfony": "^3.1.0",
     "codeception/phpunit-wrapper": "^9",
     "ergebnis/phpstan-rules": "^2.0",
-    "phpstan/phpstan": "1.10.5",
-    "phpstan/phpstan-symfony": "^1.2.20",
+    "phpstan/phpstan": "1.10.25",
```

Thank you!

This PR would be also "indirectly" resolving this task https://github.com/pimcore/pimcore/issues/15459 so would say it would be nice to continue from here.

Could you please merge all the recent changes that came afterward and bump to the latest

```suggestion
    "phpstan/phpstan": "1.10.26",
```
which just got released 2 days ago https://github.com/phpstan/phpstan/releases/tag/1.10.26 ?

---

@kingjia90 Done :)